src/Corax/IndexSearcher/IndexSearcher.cs

```diff
@@ -66,6 +69,7 @@ public IndexSearcher(Transaction tx, IndexFieldsMapping fieldsMapping = null)
         _fieldMapping = fieldsMapping ?? new IndexFieldsMapping(_transaction.Allocator);
         _fieldsTree = _transaction.ReadTree(Constants.IndexWriter.FieldsSlice);
         _metadataTree = _transaction.ReadTree(Constants.IndexMetadataSlice);
+        _persistedDynamicTreeAnalyzer = new Lazy<Tree>(() => _transaction.ReadTree(Constants.IndexWriter.DynamicFieldsAnalyzersSlice));
```

Comment:
This is single threaded, no? We can avoid an extra allocation here by just doing the check, no?

Reply:
done